### PR TITLE
Add Listener on the http endpoint

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/2783
Adds a new ServiceInstanceListener for the new HTTP port

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

### Release Note Description
Add http listener

### CODE REVIEW

File: src/Maestro/Maestro.Web/MaestroDelegatedStatelessWebService.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    protected override IEnumerable<ServiceInstanceListener> CreateServiceInstanceListeners()
+    {
+        return new[]
+        {
+            CreateServiceInstanceListener("ServiceEndpoint"),
+            CreateServiceInstanceListener("ServiceEndpointHttp")
+        };
+    }
```
Comment (premun) [inline src/Maestro/Maestro.Web/MaestroDelegatedStatelessWebService.cs:32]:
```suggestion
    protected override IEnumerable<ServiceInstanceListener> CreateServiceInstanceListeners() => new[]
    {
        CreateServiceInstanceListener("ServiceEndpoint"),
        CreateServiceInstanceListener("ServiceEndpointHttp")
    };
```

File: src/Maestro/Maestro.Web/MaestroServiceHostWebSite.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    private static void ServiceFabricMain(string serviceTypeName)
+    {
+        MaestroServiceHost.Run(host => host.RegisterStatelessWebService<TStartup>(serviceTypeName,
+            hostBuilder =>
+            {
+                hostBuilder.ConfigureAppConfiguration((context, builder) =>
+                {
+                    builder.AddDefaultJsonConfiguration(context.HostingEnvironment, serviceProvider: null);
+                });
+            }));
```
Comment (premun) [inline src/Maestro/Maestro.Web/MaestroServiceHostWebSite.cs:38]: These can be simplified to lambda expressions and everything becomes shorter?

Or maybe just the inner one would make sense to transform?

**Review by premun — COMMENTED** [review]
Much nicer!

File: src/Maestro/Maestro.Web/MaestroServiceHostWebSite.cs (hunk 2 above)
Comment (dkurepa) [reply src/Maestro/Maestro.Web/MaestroServiceHostWebSite.cs]: yup, we don't really need the curly braces

**Comment by dkurepa** [comment]
> @dkurepa where is the update from `dnceng-shared`?

https://github.com/dotnet/arcade-services/pull/2970 it's merged

**Pull request merged**